src/Services/ResponseService.php

```diff
@@ -6,6 +6,8 @@ class ResponseService
 {
     const UNEXPECTED_ERROR = 'Unexpected error';
 
+    const INVALID_CAT = 'Invalid category id';
```

Comment:
As you have now created a custom exception for this kind of error, I would store this in that exception class, pass it as the message into the exception, then pull it out of the exception to pass it into the response service